src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -640,25 +641,35 @@ class FlowCrudSpec extends HealthCheckSpecification {
 
     @Tidy
     @Tags([LOW_PRIORITY])
-    def "Unable to update to a flow with unavailable bandwidth"() {
+    def "Unable to update to a flow with #problem"() {
         given: "A flow"
         def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
         def flow = flowHelperV2.addFlow(flowHelperV2.randomFlow(srcSwitch, dstSwitch))
-        def expectedException = new ExpectedHttpClientErrorException(HttpStatus.NOT_FOUND,
-                ~/Not enough bandwidth or no path found. Switch ${srcSwitch.dpId.toString()
-                } doesn't have links with enough bandwidth, Failed to find path with requested bandwidth=${IMPOSSIBLY_HIGH_BANDWIDTH}/)
-
 
         when: "Try to update the flow "
         northboundV2.updateFlow(flow.getFlowId(),
-                flowHelperV2.toRequest(flow.tap { it.maximumBandwidth = IMPOSSIBLY_HIGH_BANDWIDTH }))
+                flowHelperV2.toRequest(flow.tap(update)))
+        println(flow.tap(update))
```

Comment:
should it be here?

Reply:
Better here, than in `develop`, but even better to be nowhere than here.
Removed.